lib/rules/at-rule-no-deprecated/__tests__/index.mjs

```diff
@@ -120,12 +111,12 @@ testRule({
 
 	reject: [
 		{
-			code: 'a { @apply foo; }',
-			message: messages.rejected('@apply'),
+			code: 'a { @nest .foo & {} }',
+			message: messages.rejected('@nest'),
```

Comment:
[question] Why did you change to the unrelated `@nest` with `@apply`?

Reply:
If you mean `@apply`by `@nest`, that's because I didn't want the reject array to be empty.